.github/workflows/build.yaml

```diff
@@ -18,13 +18,9 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Install dependencies
```

Comment:
Keep this as "install dependencies", this is still true.

Reply:
@evanna00

Reply:
I modified it just now, not pretty sure if the commands I added are correct now.

Reply:
What I meant was that you should name the section "install dependencies", but that it should use `poetry` to do so. You should remove the `pip install` commands.